demo-private.html

```diff
@@ -44,12 +45,19 @@
 </head>
 <body unresolved>
 
+<google-signin
+  clientId="1054047045356-j8pgqgls9vdef3rl09hapoicumbte0bo.apps.googleusercontent.com"
```

Comment:
This client id is required since you need a client id to authenticate. And since this demo is about being able to authenticate and use the element to access private sheets it needs to stay.